src/main/java/com/github/junrar/rarfile/FileHeader.java

```diff
@@ -197,17 +265,20 @@ public void print() {
         StringBuilder str = new StringBuilder();
         str.append("unpSize: " + getUnpSize());
         str.append("\nHostOS: " + hostOS.name());
-        str.append("\nMDate: " + mTime);
+        str.append("\nMTime: " + mTime);
```

Comment:
Could probably use double append on those, since the rest will be done in your other PR.

Reply:
Done